src/main/java/com/synopsys/integration/alert/channel/msteams/SimpleChannelActions.java

```diff
@@ -0,0 +1,14 @@
+package com.synopsys.integration.alert.channel.msteams;
+
+import com.synopsys.integration.alert.common.channel.ChannelKey;
+import com.synopsys.integration.alert.common.channel.DistributionChannel;
+
+/**
+ * This lets the DescriptorProcessor create the appropriate ChannelDistributionTestAction/ConfigurationAction combination when no customization is needed.
+ */
+public interface SimpleChannelActions {
```

Comment:
I find this name misleading. The actions are not methods of this interface, this interface marks the class as something that has "simple" actions. Something along the lines of `ActionableSimpleChannel` might get the message across better, though I'm not in love with that name either.

Reply:
This name is confusing to me especially since you have the SlackChannel extending this interface.